packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -205,6 +206,15 @@ const destructVmsPattern = pattern =>
         vms: destructPattern(pattern),
       }
 
+const checkRetentionLimit = (settings, retention) => {
```

Comment:
"check" doesn't say what it returns. Does it return `true` when it's high or low?